services/distribution/src/test/java/app/coronawarn/server/services/distribution/statistics/KeyFigureCardFactoryTest.java

```diff
@@ -261,4 +273,163 @@ private void assertKeyFigure(KeyFigure result, double value, Rank rank, Trend tr
         .containsExactly(value, rank, trend, trendSemantic, decimals);
   }
 
+  @Nested
+  @ExtendWith(SpringExtension.class)
```

Comment:
Is it possible to remove this annotations except ``@Nested``?  I think if the outer class provides this annotations then it should be enough. What do you think?